examples/example_glfw_wgpu/main.cpp

```diff
@@ -318,7 +330,10 @@ static bool InitWGPU(GLFWwindow* window)
     wgpu::Surface surface = wgpu::glfw::CreateSurfaceForWindow(instance, window);
     if (!surface)
         return false;
-    wgpu_preferred_fmt = WGPUTextureFormat_BGRA8Unorm;
+    wgpu::SurfaceCapabilities capabilities;
```

Comment:
This might be questioned, so I will explain this.
This gets the first available texture format and presentation mode for the surface, on linux current with dawn since there is no FiFo presentation mode for vulkan the example will not work for linux. 
This solves it, it will get the first available presentation mode which in most system the first one is FiFo and on linux will be Mailbox.